utils/src/main/scala/za/co/absa/enceladus/utils/testUtils/SparkTestBase.scala

```diff
@@ -17,21 +17,117 @@ package za.co.absa.enceladus.utils.testUtils
 
 import org.apache.log4j.{Level, Logger}
 import org.apache.spark.sql.SparkSession
+import org.apache.spark.SparkConf
+import com.typesafe.config.ConfigFactory
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{Path, FileSystem}
+import scala.collection.JavaConversions._
+import java.io.File
 import za.co.absa.enceladus.utils.time.TimeZoneNormalizer
+import com.typesafe.config.Config
 
-trait SparkTestBase {
-  System.setProperty("user.timezone", "UTC");
+trait SparkTestBase { self =>
+  System.setProperty("user.timezone", "UTC")
```

Comment:
This setup shouldn't be needed considering few lines later the `TimeZoneNormalizer` is used. (I know it was there before, probably overlooked)

Reply:
Cool, getting rid of it